massa-execution-worker/src/interface_impl.rs

```diff
@@ -1066,8 +1070,11 @@ impl Interface for InterfaceImpl {
     /// # Arguments:
     /// data: the bytes_array data that is the payload of the event
     fn generate_event_wasmv1(&self, data: Vec<u8>) -> Result<()> {
-        let data_str = String::from_utf8(data.clone()).unwrap_or(format!("{:?}", data));
+        if data.len() > self.config.max_event_size {
```

Comment:
I bet it should be used in many other places.
So I'd make it an helper function :
```rust
fn check (len: usize) -> Result<()> {}
// or
fn check_len (data: &[u8]) -> Result<()> {}
```
so it can be called 
`check(data.len())?;`

Reply:
The helper function would also need the limit passed as parameter which is nearly the same as making the check directly in the function.